rich: add page (#8044)
tldr-pages/tldr: pages/common/rich.md

# rich

> Rich CLI - fancy terminal display.
> More information: <https://github.com/Textualize/rich-cli>.

> - Show file contents with syntax highlighting:

Reviewer: ```suggestion
- Show a specific file with syntax highlighting:
```

Reviewer: @navarroaxel, what about to stick to `Display` word instead of `Print` in all pages?

> `rich file.py`

Reviewer 2: ```suggestion
`rich {{path/to/file.py}}`
```

- Add line numbers, and indentation guides:

> `rich file.py --line-number --guides`

Reviewer 2: ```suggestion
`rich {{path/to/file.py}} --line-number --guides`
```

> - Apply a theme:

Reviewer: ```suggestion
- Apply a specific theme:
```

> `rich file.py --theme monokai`

Reviewer 2: ```suggestion
`rich {{path/to/file.py}} --theme {{monokai}}`
```

- Display file in an interactive pager:

> `rich file.py --pager`

Reviewer 2: ```suggestion
`rich {{path/to/file.py}} --pager`
```

> - Display file in an interactive pager:

Reviewer: ```suggestion
- Display a specific file in an interactive pager:
```

> - Display contents from a URL:

Reviewer: ```suggestion
- Display contents from a specific URL:
```

> `rich https://raw.githubusercontent.com/Textualize/rich-cli/main/README.md --markdown --pager`

Reviewer 2: ```suggestion
`rich {{https://raw.githubusercontent.com/Textualize/rich-cli/main/README.md}} --markdown --pager`
```

- Export a file as HTML:

> `rich file.md -o file.html`

Reviewer 2: ```suggestion
`rich {{path/to/file.md}} -o {{path/to/file.html}}`
```

> - Export a file as HTML:

Reviewer: ```suggestion
- Export a specific file as HTML:
```

> `rich {{path/to/file.md}} -o {{path/to/file.html}}`

Reviewer 3: We prefer long arguments if available
```suggestion
`rich {{path/to/file.md}} --export-html {{path/to/file.html}}`
```

> - Print text with formatting tags, custom alignment, and fixed line width:

Reviewer: ```suggestion
- Display a text with specific formatting tags, alignment, and line width:
```

> `rich --print "Hello [green on black]Stylized[/green on black] [bold]World[/bold]" --center --width 10`

Reviewer 2: ```suggestion
`rich --print {{"Hello [green on black]Stylized[/green on black] [bold]World[/bold]"}} --center --width {{10}}`
```

> `rich --print {{"Hello [green on black]Stylized[/green on black] [bold]World[/bold]"}} --center --width {{10}}`

Reviewer: ```suggestion
`rich --print {{"Hello [green on black]Stylized[/green on black] [bold]World[/bold]"}} --{{left|center|right}} --width {{10}}`
```
